app-react/src/main/kotlin/com/ustadmobile/view/ContentEntryDetailOverviewScreen.kt

```diff
@@ -0,0 +1,483 @@
+import react.FC
+import react.Props
+import react.create
+import react.useState
+
+external interface ContentEntryDetailOverviewScreenProps : Props {
+
+    var uiState: ContentEntryDetailOverviewUiState
+
+    var onClickDownload: () -> Unit
+
+    var onClickOpen: () -> Unit
+
+    var onClickMarkComplete: () -> Unit
+
+    var onClickDelete: () -> Unit
+
+    var onClickManageDownload: () -> Unit
+
+    var onClickTranslation: () -> Unit
+
+    var onClickContentJobItem: () -> Unit
+}
+
+val ContentEntryDetailOverviewComponent2 = FC<ContentEntryDetailOverviewScreenProps> { props ->
+
+    val strings: StringsXml = useStringsXml()
+
+    Container {
+        Stack {
+            spacing = responsive(20.px)
+
+            ContentDetails{
+                uiState = props.uiState
+            }
+
+            if (props.uiState.contentEntryButtons?.showDownloadButton == true){
+                Button{
+                    variant = ButtonVariant.contained
+                    onClick = { props.onClickDownload }
+
+                    + strings[MessageID.download].uppercase()
+                }
+            }
+
+            if (props.uiState.contentEntryButtons?.showOpenButton == true){
+                Button {
+                    onClick = { props.onClickOpen }
+                    variant = ButtonVariant.contained
+
+                    + strings[MessageID.open].uppercase()
+                }
+            }
+
+            ContentJobList{
+                uiState = props.uiState
+            }
+
+            if (props.uiState.locallyAvailable) {
+                LocallyAvailableRow()
+            }
+
+            Divider { orientation = Orientation.horizontal }
+
+            QuickActionBarsRow {
+                uiState = props.uiState
+                onClickMarkComplete = props.onClickMarkComplete
+                onClickDelete = props.onClickMarkComplete
+                onClickManageDownload =  props.onClickMarkComplete
+            }
+
+            Typography{
+                + (props.uiState.contentEntry?.description ?: "")
+            }
+
+            Divider { orientation = Orientation.horizontal }
+
+            if (props.uiState.translationVisibile){
+                Typography{
+                    + strings[MessageID.also_available_in]
+                }
+
+                Grid {
+                    direction = responsive(GridDirection.row)
+                    container = true
+
+                    props.uiState.availableTranslations.forEach {
+                        Grid {
+                            item = true
+                            xs = 2
+                            md = 1
+
+                            Box {
+                                sx {
+                                    padding = 8.px
+                                    textAlign = TextAlign.center
+                                }
+
+                                Button {
+                                    onClick = { props.onClickTranslation }
```

Comment:
Needs variant = ButtonVariant.text to show the correct button style (e.g. textbutton), should not have background color.